File: test/fixtures/typed.ts
function log(msg: string | number): void {
  console.log(msg);
}

const greeting: string = "hello";
const count: number = 42;
const enabled: boolean = true;

function caller(s: string, n: number, b: boolean) {
  log(s);
  log(n);
  log("inline string");
  log(123);
  log(greeting);
  log(count);
  log(enabled as any);
}

File: test/fixtures/typed.go
package main

import "fmt"

func log(msg string) {
	fmt.Println(msg)
}

var greeting string = "hello"
var count int = 42

func caller(s string, n int) {
	log(s)
	log("inline")
	log(greeting)
	log(count)
}
